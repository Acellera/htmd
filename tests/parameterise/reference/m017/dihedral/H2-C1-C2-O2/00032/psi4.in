set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 1.550054	 -0.106184	 -1.858756
O	 0.729007	 -0.205652	 -1.398857
C	 0.702312	 0.627605	 -0.264544
C	 -0.714990	 0.596302	 0.266387
H	 0.967556	 1.647356	 -0.530264
H	 1.396756	 0.274779	 0.494921
O	 -1.405996	 -0.526611	 -0.195820
H	 -1.257453	 1.468809	 -0.075599
H	 -0.688972	 0.618407	 1.353696
H	 -0.890587	 -1.292708	 0.022039

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		10 7 4 3
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "10  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
