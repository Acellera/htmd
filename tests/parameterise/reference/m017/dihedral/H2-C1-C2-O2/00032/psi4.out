    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 16:23:22 2016

    Process ID:  25757
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 1.550054	 -0.106184	 -1.858756
O	 0.729007	 -0.205652	 -1.398857
C	 0.702312	 0.627605	 -0.264544
C	 -0.714990	 0.596302	 0.266387
H	 0.967556	 1.647356	 -0.530264
H	 1.396756	 0.274779	 0.494921
O	 -1.405996	 -0.526611	 -0.195820
H	 -1.257453	 1.468809	 -0.075599
H	 -0.688972	 0.618407	 1.353696
H	 -0.890587	 -1.292708	 0.022039

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		10 7 4 3
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "10  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-49-4-2.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 16:23:23 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           H          1.709551849734    -0.196538235225    -1.438308364270     1.007825032070
           O          0.888504849734    -0.296006235225    -0.978409364270    15.994914619560
           C          0.861809849734     0.537250764775     0.155903635730    12.000000000000
           C         -0.555492150266     0.505947764775     0.686834635730    12.000000000000
           H          1.127053849734     1.557001764775    -0.109816364270     1.007825032070
           H          1.556253849734     0.184424764775     0.915368635730     1.007825032070
           O         -1.246498150266    -0.616965235225     0.224627635730    15.994914619560
           H         -1.097955150266     1.378454764775     0.344848635730     1.007825032070
           H         -0.529474150266     0.528052764775     1.774143635730     1.007825032070
           H         -0.731089150266    -1.383062235225     0.442486635730     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.47956  B =      0.21834  C =      0.16277 [cm^-1]
  Rotational constants: A =  14376.85763  B =   6545.57936  C =   4879.63560 [MHz]
  Nuclear repulsion =  135.863215837096647

  Charge       = 0
  Multiplicity = 1
  Electrons    = 34
  Nalpha       = 17
  Nbeta        = 17

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 36
    Number of basis function: 72
    Number of Cartesian functions: 72
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A         72      72       0       0       0       0
   -------------------------------------------------------
    Total      72      72      17      17      17       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 150
    Number of basis function: 474
    Number of Cartesian functions: 474
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 4.0126750315E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -200.29376278418064   -2.00294e+02   8.47282e-02 
   @DF-RHF iter   2:  -188.99412534032737    1.12996e+01   7.78368e-02 DIIS
   @DF-RHF iter   3:  -223.10937165112668   -3.41152e+01   4.22623e-02 DIIS
   @DF-RHF iter   4:  -228.37885277476269   -5.26948e+00   1.39855e-02 DIIS
   @DF-RHF iter   5:  -228.83496384287295   -4.56111e-01   5.79420e-03 DIIS
   @DF-RHF iter   6:  -228.90316585075610   -6.82020e-02   1.50757e-03 DIIS
   @DF-RHF iter   7:  -228.90938548674521   -6.21964e-03   5.56080e-04 DIIS
   @DF-RHF iter   8:  -228.91027582339061   -8.90337e-04   1.29920e-04 DIIS
   @DF-RHF iter   9:  -228.91033206599835   -5.62426e-05   4.41858e-05 DIIS
   @DF-RHF iter  10:  -228.91033876759442   -6.70160e-06   1.27015e-05 DIIS
   @DF-RHF iter  11:  -228.91033935660684   -5.89012e-07   2.16493e-06 DIIS
   @DF-RHF iter  12:  -228.91033938018845   -2.35816e-08   7.00461e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.559413     2A    -20.545375     3A    -11.275152  
       4A    -11.267383     5A     -1.378403     6A     -1.342439  
       7A     -1.002035     8A     -0.840912     9A     -0.715505  
      10A     -0.688563    11A     -0.647742    12A     -0.589566  
      13A     -0.573916    14A     -0.527983    15A     -0.484994  
      16A     -0.441297    17A     -0.430728  

    Virtual:                                                              

      18A      0.210077    19A      0.260485    20A      0.272077  
      21A      0.282323    22A      0.320110    23A      0.354649  
      24A      0.385300    25A      0.433434    26A      0.477940  
      27A      0.748570    28A      0.789132    29A      0.804800  
      30A      0.826050    31A      0.905198    32A      0.970788  
      33A      1.050340    34A      1.075435    35A      1.116677  
      36A      1.128021    37A      1.154759    38A      1.174097  
      39A      1.203673    40A      1.215330    41A      1.236346  
      42A      1.319246    43A      1.334770    44A      1.401550  
      45A      1.479129    46A      1.592021    47A      1.689604  
      48A      1.743210    49A      1.825592    50A      1.980728  
      51A      1.996044    52A      2.058959    53A      2.139492  
      54A      2.159195    55A      2.227636    56A      2.378330  
      57A      2.383906    58A      2.483026    59A      2.523953  
      60A      2.638156    61A      2.685179    62A      2.732383  
      63A      2.813936    64A      2.866152    65A      2.909403  
      66A      3.074753    67A      3.171754    68A      3.260890  
      69A      4.139380    70A      4.332362    71A      4.607356  
      72A      4.876668  

    Final Occupation by Irrep:
              A 
    DOCC [    17 ]

  Energy converged.

  @DF-RHF Final Energy:  -228.91033938018845

   => Energetics <=

    Nuclear Repulsion Energy =            135.8632158370966465
    One-Electron Energy =                -582.9244630661280553
    Two-Electron Energy =                 218.1509078488430191
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -228.9103393801884181

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     1.9054      Y:     1.9346      Z:     1.8045

  Electronic Dipole Moment: (a.u.)
     X:    -0.7325      Y:    -1.5437      Z:    -1.3491

  Dipole Moment: (a.u.)
     X:     1.1729      Y:     0.3909      Z:     0.4554     Total:     1.3176

  Dipole Moment: (Debye)
     X:     2.9813      Y:     0.9937      Z:     1.1574     Total:     3.3489


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-49-4-2.cx1.hpc.ic.ac.uk at Tue Jul  5 16:23:25 2016
Module time:
	user time   =       2.23 seconds =       0.04 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          2 seconds =       0.03 minutes
Total time:
	user time   =       2.23 seconds =       0.04 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          2 seconds =       0.03 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     1.9054      Y:     1.9346      Z:     1.8045

  Electronic Dipole Moment: (a.u.)
     X:    -0.7325      Y:    -1.5437      Z:    -1.3491

  Dipole Moment: (a.u.)
     X:     1.1729      Y:     0.3909      Z:     0.4554     Total:     1.3176

  Dipole Moment: (Debye)
     X:     2.9813      Y:     0.9937      Z:     1.1574     Total:     3.3489

  Quadrupole Moment: (Debye Ang)
    XX:   -24.8092     YY:   -22.2288     ZZ:   -23.8725
    XY:    -0.6255     XZ:    -2.2267     YZ:    -1.1850

  Traceless Quadrupole Moment: (Debye Ang)
    XX:    -1.1724     YY:     1.4081     ZZ:    -0.2357
    XY:    -0.6255     XZ:    -2.2267     YZ:    -1.1850

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     H     0.28010  0.28010  0.00000  0.43980
       2     O     4.36457  4.36457  0.00000 -0.72914
       3     C     3.02032  3.02032  0.00000 -0.04063
       4     C     2.99783  2.99783  0.00000  0.00433
       5     H     0.42521  0.42521  0.00000  0.14959
       6     H     0.42974  0.42974  0.00000  0.14052
       7     O     4.35823  4.35823  0.00000 -0.71647
       8     H     0.41397  0.41397  0.00000  0.17205
       9     H     0.43013  0.43013  0.00000  0.13974
      10     H     0.27989  0.27989  0.00000  0.44023

   Total alpha = 17.00000, Total beta = 17.00000, Total charge = -0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
